docs/pages/features/chakra-factory.md

```diff
@@ -0,0 +1,112 @@
+# Chakra Factory
+
+Chakra factory serves as an **object of chakra enabled JSX elements**, and also
+**a function that can be used to enable custom component** to receive chakra's
+style props.
+
+```bash
+import { chakra } from "@chakra-ui/vue-next"
+```
+
+## Chakra JSX Elements
+
+Create base HTML elements with theme-aware style props using `chakra.<element>`
+notation. For example, if you want a plain HTML button with the ability to pass
+chakra styles, you can write `<chakra.button />`.
+
+```jsx
+<chakra.button
+  px="3"
+  py="2"
+  bg="green.200"
+  rounded="md"
+  :_hover="{ bg: "green.300" }"
+>
+  Click me
+</chakra.button>
+```
```

Comment:
I think we can directly use Vue SFCs for the docs codeblocks

```suggestion
```vue
<chakra.button
  px="3"
  py="2"
  bg="green.200"
  rounded="md"
  :_hover="{ bg: 'green.300' }"
>
  Click me
</chakra.button>
```
```